contracts/arbos/builtin/ArbGasInfo.sol

```diff
@@ -10,10 +10,10 @@ interface ArbGasInfo {
     //            per ArbGas congestion,
     //            per ArbGas total
     //        )
-    function getPricesInWei() external view returns (uint, uint, uint, uint, uint, uint);
+    function getPricesInWei(address aggregator) external view returns (uint, uint, uint, uint, uint, uint);
 
     // return prices in ArbGas (per L2 tx, per L1 calldata unit, per storage allocation)
-    function getPricesInArbGas() external view returns (uint, uint, uint);
+    function getPricesInArbGas(address aggregator) external view returns (uint, uint, uint);
```

Comment:
We should discuss why types of API consistency guarentees we should provide. Theoretically this change could break deployed contracts though it seems unlikely

Reply:
I'm going to change this to leave the old no-argument method in place and add a new method that takes an aggregator address argument.  The no-arg method will call the new method, passing the address of the sender's preferred aggregator.